index.html

```diff
@@ -5917,6 +5923,7 @@ <h4>JsonLdErrorCode</h4>
             "invalid value object",
             "invalid value object value",
             "invalid vocab mapping",
+            "iri confused with prefix",
```

Comment:
Match test and capitalization in other codes.
```suggestion
            "IRI confused with prefix",
```

Reply:
Did through separate commit, as there was a third.